README.md

```diff
@@ -3,3 +3,15 @@
 A collection of resources - both custom made and links to other sites - to help people understand what goes on under the hood when using git, and provide information about useful git commands.
 
 You can chat with us [in this gitter chat room](https://gitter.im/mindfulness-at-the-computer/git-info) if you want to ask questions about git, make suggestions for improvements, ask for clarifications, etc.
+
+## Common git commands
+
+**To be completed**
+
+## Adavanced git commands
+
+**To be completed**
+
+## Simple edit/push/pull-request workflow for fork/clone development
```

Comment:
I think this would be valuable, i've gotten some feedback on the workflow overview i created in draw.io and it's been very positive, and for me personally i feel less anxious using git with an overview by my side :)

If we create a similar thing in this repo we could perhaps remove references to mindfulness-at-the-computer (not sure how closely we want this repo to relate to the other)